packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -235,6 +239,13 @@ class CupertinoTextField extends StatefulWidget {
   /// main text entry except a lighter font weight and a grey font color.
   final String placeholder;
 
+  /// The style to use for the placeholder text which overrides [style] property.
+  ///
+  /// If null, placeholder's style will be the same as [style].
```

Comment:
This isn't true, is it?  the placeholder should at least have a special color even if `placeholderStyle` isn't provided, right?